### PR TITLE
Fix heading offset

### PR DESCRIPTION
**Pull request by ctzsnooze**
The heading value shown in Explorer's IMU calculation did not match that shown in the GPS Heading or IMU Debug values.  

I noted that the blackbox IMU heading calculation is derived from other parameters, and differs from the IMU heading value in debugs.  However it may be useful in some cases, and should at least have the same values and display the same when graphed.  This PR fixes that problem so that all heading values match up numerically and in graphs.

Other minor changes:

- changed 'deg' to the degree symbol, consistently
- put a space before units, consistently, for clarity
- removed unnecessary whitespace
- added Altitude graph configuration
- added a small change to RTH to graph display values, which can be reversed once debugging of maxAlt is complete.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [reply js/flightlog_fields_presenter.js:855]: Same for line 860, 865, 897, 1011, 1023, 1025

Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: thanks, fixed

File: js/imu.js
Comment (haslinghuis) [inline js/imu.js:129]:
```suggestion
            if (heading > 2 * Math.PI)
               heading -= 2 * Math.PI;
```

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:1044]:
```suggestion
                            return value.toFixed(0);
```

File: js/graph_config.js
Comment (haslinghuis) [inline js/graph_config.js:925]:
```suggestion
        const sysConfig = flightLog.getSysConfig();
        const debugModeName = DEBUG_MODE[sysConfig.debug_mode];
```

File: js/imu.js
Comment (haslinghuis) [inline js/imu.js:130]:
```suggestion
            if (heading > 2 * Math.PI) {
               heading -= 2 * Math.PI;
            }

```

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:1044]: Thanks, that was a silly mistake.

File: js/graph_config.js
Comment (ctzsnooze) [reply js/graph_config.js:925]: This change was from a commit that was accidentally added in this PR by mistake.  

It was intended to add the debug name into the text that appears to the right of the graph, but it didn't work, so I've removed that commit.

That removes the need to make this change, but I'll put it into the code I was testing.

Thank you again.

File: js/imu.js
Comment (ctzsnooze) [reply js/imu.js:130]: thanks, wasn't sure of the need for the braces

**Comment by ctzsnooze** [comment]
Thanks very much @haslinghuis for your advice and comments.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
